src/hearings/components/hearing-parties-title/hearing-parties-title.component.html

```diff
@@ -0,0 +1 @@
+<span class="govuk-caption-l">Request a hearing for TEST_CONENT1 vs TEST_CONENT2</span>
```

Comment:
If we could use the same text content in the prototype please,

Reply:
updated content to prototype